### PR TITLE
Admin clients redux (branched off Redux Update for Teams Clients Employees)

### PR DESCRIPTION
**Pull request by paulmatchett**
## Changes made

- Branched off teams-clients-employees-redux branch
- ViewClients now using redux to display data.
- CreateClient now using redux to post data.

### CODE REVIEW

File: src/components/AllClients/container.js [hunk 1]
```diff
@@ -1,49 +1,62 @@
...
       if (shouldRefresh) {
-        this.getClients();
+        this.props.fetchAllClients();
       }
     };
 
     render() {
+
+      const { clients, isUpdating, selectedClientId } = this.props;
+      let selectedClient = clients.filter(({clientId}) => clientId === selectedClientId)[0];
```
Comment (bgribz) [inline src/components/AllClients/container.js]: `Array.find()` would be better to use in this situation where we're after a single element.

```
let selectedClient = clients.find(({clientId}) => clientId === selectedClientId);
```

Comment (paulmatchett) [reply src/components/AllClients/container.js]: Nice 👍 This has been updated

File: src/components/CreateClient/container.js [hunk 2]
```diff
@@ -12,8 +13,16 @@ const CreateClientContainer = Wrapped =>
...
+      resetForm();
+      Toast({
+        type: 'success',
+        title: `${clientName} created sucessfully! 👍`,
+      });
+    }
+
     submitRequest = (data, resetForm) => {
-      this.props.postNewClient(data, resetForm);
+      this.props.postNewClient(data, this.onSuccess(data.clientName, resetForm));
```
Comment (William-Unosquare) [inline src/components/CreateClient/container.js]: () => this.onSuccess

Comment (paulmatchett) [reply src/components/CreateClient/container.js]: 👍

**Pull request merged**